lib/logstash/outputs/influxdb.rb

```diff
@@ -20,7 +20,10 @@ class LogStash::Outputs::InfluxDB < LogStash::Outputs::Base
   config_name "influxdb"
 
   # The database to write
-  config :db, :validate => :string, :default => "stats"
+  config :db, :validate => :string, :default => "statistics"
```

Comment:
This db change seems like a backwards-incompatible change. How will users react to this? Is the change to the default required?